src/components/manage/Blocks/Search/components/SelectStyling.jsx

```diff
@@ -32,6 +32,7 @@ export const customSelectStyles = {
   valueContainer: (styles) => ({
     ...styles,
     padding: '0px',
+    paddingTop: '4px',
     paddingLeft: 0,
```

Comment:
@dobri1408  this is redundant given that at line 34 you have padding 0

Reply:
@dobri1408 this doesn't fix the fact that you can also have bottom text clipping when you use p or g
see the screenshot
![4px-2-rows-demo](https://github.com/plone/volto/assets/152852/05a7abd8-5fff-4afa-89a6-a7cc93ed15e2)